### PR TITLE
update(logos): adds updated logo from legal and adds color variations

### PR DESCRIPTION
**Pull request by technabors**
@CITguy received updated logo assets (from Marketing) after discussing with the legal team. This PR adds links to the updated logos / color variations.

### Before
image

### After
image

### CODE REVIEW

**Comment by CITguy** [comment]
@nimbinatus, FYI: these images should not be embedded, because we shouldn't need to override their styling. They can use a plain old `` tag.

**Comment by CITguy** [comment]
Dev LGTM

File: _docs/style/iconography.md [hunk 1]
```diff
@@ -176,24 +176,48 @@ Rackspace US, Inc. retains copyright and/or applicable rights to all text and gr
 
 {% column right:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-{% figure [caption: "<a target='_blank' href='http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/fanatiguy.svg'>Fanatiguy</a>"][class:"image bg-light border"] %}
-![](http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/fanatiguy.svg){:width="100px"}
+{% figure [caption: "<a target='_blank' href='http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/fanatiguy-black.svg'>Fanatiguy - Black</a>"][class:"image bg-light border"] %}
```
Comment (nimbinatus) [inline _docs/style/iconography.md]: @technabors, can you add a comment above this to not use the `embed` tag for when we go through the repo to make SVGs happy?

File: _docs/style/iconography.md [hunk 2]
```diff
@@ -176,24 +176,48 @@ Rackspace US, Inc. retains copyright and/or applicable rights to all text and gr
...
+![](http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/fanatiguy-black.svg){:width="100px"}
 {% endfigure %}
 
 {% endcolumn %}
 
 {% column right:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-{% figure [caption:"<a target='_blank' href='http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/rs-logo.svg'>Rackspace logo - Gray</a>"][class:"image bg-light border center"] %}
-![](http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/rs-logo.svg){:width="286px"}
+{% figure [caption: "<a target='_blank' href='http://c1ee333499ed5f44e56a-fa12562cfe810d69bedcc36a0ac289ef.r55.cf1.rackcdn.com/img/fanatiguy-color.svg'>Fanatiguy - Color</a>"][class:"image bg-light border"] %}
```
Comment (nimbinatus) [inline _docs/style/iconography.md]: Same here

Comment (nimbinatus) [reply _docs/style/iconography.md]: and on down, actually

**Comment by technabors** [comment]
@araiford Bit of irony here; removing the `center` class from the corporate logos actually centered them correctly. It was required when it was in the same row as the fanatiguy but actually uncentered them once it was in its own row.

image

File: _docs/style/iconography.md [hunk 3]
```diff
@@ -175,25 +175,49 @@ Rackspace US, Inc. retains copyright and/or applicable rights to all text and gr
 
 
 {% column right:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+// Do not use the embed tag on this image when we refactor SVGs
```
Comment (nimbinatus) [inline _docs/style/iconography.md]: Switch to `<!--` and `-->` for comments

screen shot 2018-04-04 at 4 14 40 pm

Comment (technabors) [reply _docs/style/iconography.md]: ow gosh that was silly of me. updating.

**Pull request merged**